prusti-contracts/prusti-specs/src/specifications/mod.rs

```diff
@@ -11,7 +11,7 @@
 ///           ;
 /// requires ::= "requires", "(", prusti_expr, ")" ;
 /// ensures ::= "ensures", "(", prusti_expr, ")" ;
-/// 
+///
 /// This grammar doesn't yet handle the unintuitive precedence difference between `&&` and `||` operators.
```

Comment:
(This entire comment is outdated now.)

Reply:
Not just because of this PR though, right? Should I just put down a FIXME or something?

Reply:
No, I just noticed because of the reformat. But feel free to just remove the comment, it's not valuable if it's not accurate anyway.